app/server.rb

```diff
@@ -0,0 +1,28 @@
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, 'super secret'
+
+
+
+  register Sinatra::Flash
+  register Sinatra::Partial
+
+  set :partial_template_engine, :erb
+  enable :partial_underscores
+
+  use Rack::MethodOverride
+
+  set :root, File.dirname(__FILE__)
+  set :views, Proc.new { File.join(root, "views") }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)